.dotnet/src/Custom/FineTuning/FineTuningClient.Protocol.cs

```diff
@@ -224,4 +224,66 @@ public virtual ClientResult GetJobEvents(string jobId, string after, int? limit,
         using PipelineMessage message = CreateGetFineTuningEventsRequest(jobId, after, limit, options);
         return ClientResult.FromResponse(_pipeline.ProcessMessage(message, options));
     }
+
+    /// <summary>
+    /// [Protocol Method] List the checkpoints for a fine-tuning job.
+    /// <list type="bullet">
+    /// <item>
+    /// <description>
+    /// This <see href="https://aka.ms/azsdk/net/protocol-methods">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
+    /// </description>
+    /// </item>
+    /// <item>
+    /// <description>
+    /// Please try the simpler <see cref="GetFineTuningJobCheckpointsAsync(string,string,int?)"/> convenience overload with strongly typed models first.
```

Comment:
I think this part of the doc comments that is auto-generated for protocol methods is a problem. In most cases, we have renamed the methods. Here, in fine-tuning, the convenience overload doesn't even exist yet. 🙃 Conseuqently, I've been deleting this part of the doc comment for protocol methods as you can see above.

Reply:
Good call -- I'll remove these `<list>` elements across all new protocol additions.